src/core.jl

```diff
@@ -104,4 +104,27 @@ function match_keypoints(keypoints_1::Keypoints, keypoints_2::Keypoints, desc_1,
         end
     end
     matches
+end
+
+"""
+```
+grade = grade_matches(keypoints_1, keypoints_2, difference_method)
+```
+
+Returns a measure of similarity between keypoints in `keypoints_1` and `keypoints_2` where `difference_method`
+is the method used for computing difference between individual pair of keypoints.
+"""
+
+function grade_matches(K1::Keypoints, K2::Keypoints, diff::Function = (i,j)->(abs(i[1] - j[1]) + (abs(i[2] - j[2]))))
```

Comment:
A nice addition to this would be a version of this function which takes in a `limit` parameter and returns the percentage of keypoints which have distance under this limit.